pkg/cli/upgradeassistant/cmd/migrate/180.go

```diff
@@ -0,0 +1,91 @@
+package migrate
+
+import (
+	"context"
+	"fmt"
+
+	"go.mongodb.org/mongo-driver/bson"
+
+	"github.com/koderover/zadig/pkg/cli/upgradeassistant/internal/upgradepath"
+	"github.com/koderover/zadig/pkg/config"
+	"github.com/koderover/zadig/pkg/microservice/policy/core/repository/models"
+	"github.com/koderover/zadig/pkg/microservice/policy/core/repository/mongodb"
+	"github.com/koderover/zadig/pkg/setting"
+	"github.com/koderover/zadig/pkg/tool/log"
+	mongotool "github.com/koderover/zadig/pkg/tool/mongo"
+)
+
+func init() {
+	upgradepath.AddHandler(upgradepath.V171, upgradepath.V180, V171ToV180)
+	upgradepath.AddHandler(upgradepath.V180, upgradepath.V171, V180ToV171)
+}
+
+// V171ToV180 update all the roleBinding names in this format "{uid}-{roleName}-{roleNamespace}"
+// Caution: this migration contains unrecoverable changes, please back up the database in advance
+func V171ToV180() error {
+	log.Info("Migrating data from 1.7.1 to 1.8.0")
+
+	if err := updateAllRoleBindingNames(); err != nil {
+		log.Errorf("Failed to update roleBinding names, err: %s", err)
+		return err
+	}
+
+	return nil
+}
+
+func V180ToV171() error {
```

Comment:
This rollback operation didn't work, why do we need to provide it?

Reply:
it is a noop, to ensure the `helm rollback` not break. the whole progress is: 1. restore the database; 2. run `helm rollback`